Dockerfile

```diff
@@ -87,6 +87,9 @@ COPY --from=0 /usr/local/bin/kapp .
 # Name it kapp-controller to identify it easier in process tree
 COPY --from=0 /go/src/github.com/vmware-tanzu/carvel-kapp-controller/controller kapp-controller
 
+# Copy the ca-bundle so we have an original
+RUN cp /etc/pki/tls/certs/ca-bundle.crt /etc/pki/tls/certs/ca-bundle.crt.orig
```

Comment:
I chose `RUN cp` over `COPY` because `COPY` was failing with a "file not found error". Seems like something to do with accessing the `/etc/` folder via the `COPY` syntax.